### ISSUE TITLE
Monotif Test drop generating various previews on refresh

### ISSUE THREAD

**Issue by prury** [kodadot/nft-gallery#9648]
https://canary.kodadot.xyz/ahk/drops/monotifs

https://github.com/kodadot/nft-gallery/assets/36627808/8386401f-86a2-489d-9630-61155125f3a6

**Comment by exezbcz** [comment]
yes happened to me twice but could not reproduce consistently, thanks for reporting
